roles/node_exporter/templates/node_exporter.service.j2

```diff
@@ -42,6 +42,9 @@ StartLimitInterval=0
 {% for m in ansible_mounts if m.mount.startswith('/home') %}
 {%   set ns.protect_home = 'read-only' %}
 {% endfor %}
```

Comment:
Doesn't this serve the same purpose as the proposed change?

Reply:
Ok can we then remove this in favor of the proposed change or is there a scenario where the mountpoint `/home` needs to have protecthome without the textfile being there?

Reply:
Aha gotcha!